modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,48 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
```

Comment:
We could change it for:
For example, suppose that an API specification contained multiple examples within a response:

Reply:
Changed.